terraform/main.tf

```diff
@@ -98,3 +98,255 @@ resource "google_storage_bucket_iam_binding" "binding" {
     "user:aengel@rmi.org",
   ]
 }
+
+  resource "google_artifact_registry_repository" "pudl-superset-repo" {
+    location = "us-central1"
+    repository_id = "pudl-superset"
+    description = "Docker image of PUDL superset deployment."
+    format = "docker"
+  }
+
+resource "google_cloud_run_v2_service" "default" {
+  name     = "pudl-superset"
+  location = "us-central1"
+  client   = "terraform"
+
+  template {
+    containers {
+      image = "us-central1-docker.pkg.dev/catalyst-cooperative-pudl/pudl-superset/pudl-superset:latest"
+
+      volume_mounts {
+        name       = "cloudsql"
+        mount_path = "/cloudsql"
+      }
+
+      env {
+        name = "SUPERSET_DB_USER"
+        value_source {
+          secret_key_ref {
+            secret = "superset-database-username"
+            version = "1"
+          }
+        }
+      }
+      env {
+        name = "SUPERSET_DB_NAME"
+        value_source {
+          secret_key_ref {
+            secret = "superset-database-database"
+            version = "1"
+          }
+        }
+      }
+      env {
+        name = "SUPERSET_DB_PASS"
+        value_source {
+          secret_key_ref {
+            secret = "superset-database-password"
+            version = "1"
+          }
+        }
+      }
+      env {
+        name = "SUPERSET_SECRET_KEY"
+        value_source {
+          secret_key_ref {
+            secret = "superset-secret-key"
+            version = "1"
+          }
+        }
+      }
+      env {
+        name = "CLOUD_SQL_CONNECTION_NAME"
+        value_source {
+          secret_key_ref {
+            secret = "superset-database-connection-name"
+            version = "1"
+          }
+        }
+      }
+      env {
+        name = "IS_CLOUD_RUN"
+        value = "True"
+      }
+      env {
+        name = "AUTH0_CLIENT_ID"
+        value_source {
+          secret_key_ref {
+            secret = "superset-auth0-client-id"
+            version = "1"
+          }
+        }
+      }
+      env {
+        name = "AUTH0_CLIENT_SECRET"
+        value_source {
+          secret_key_ref {
+            secret = "superset-auth0-client-secret"
+            version = "1"
+          }
+        }
+      }
+      env {
+        name = "AUTH0_DOMAIN"
+        value_source {
+          secret_key_ref {
+            secret = "superset-auth0-domain"
+            version = "1"
+          }
+        }
+      }
+
+      ports {
+        container_port = 8088
+      }
+      resources {
+        limits = {
+          cpu    = "4"
+          memory = "2048Mi"
+        }
+      }
+    }
+    volumes {
```

Comment:
Did something come up with the GCS volume mount stuff, or is that just not pushed yet?

Reply:
I hadn't pushed it!